.github/workflows/lint.yaml

```diff
@@ -99,6 +99,6 @@ jobs:
         run: pip install -r requirements-dev.txt
 
       - name: Run black
-        run: black . --check --config black-config.toml
+        run: black --check --include "(tests|sripts)" .
```

Comment:
```suggestion
        run: black --check tests scripts
```

Reply:
`scripts` is not there yet – won't work.